documents/contributing.md

```diff
@@ -0,0 +1,79 @@
+      meaningful name, such as `fix-typos-in-select-query` or `add-groupby`.
+   4. Make your changes, [commit them, and push them to your repository on  GitHub](https://help.github.com/articles/create-a-repo/#commit-your-first-change).
+   5.  Send a [pull request](https://help.github.com/articles/using-pull-requests/) to the `gh-pages` branch of the repository
+
+- From GitHub.com
+   1. Click on the `Fork` button at the top right corner of the [`datacarpentry/semester-biology` repository on GitHub](https://github.com/datacarpentry/semester-biology).
+   2. Navigate to your forked repository at `https://yourusername.github.io/semester-biology/`
+   3. Navigate to the file you want to change (*like [`contibuting.md`](https://github.com/datacarpentry/semester-biology/blob/gh-pages/documents/contributing.md)*) and click on the `Edit this file` button. *It's a pencil!*
+   4. Make changes to the file.
+   5. Commit the changes using the form at the bottom of the `edit` page. If you are working on your own forked version of the course, you can choose 'Commit directly to the `gh-pages` branch'. The other option ('Create a **new branch**') is used for a work flow with [Pull Requests](https://help.github.com/articles/using-pull-requests), which is our preferred way of receiving collaborative contributions.
+
+- If it is easier for you to send your changes to us some other way, please mail 
+us at _LINK TO EMAIL_. Given a choice between you creating content or wrestling 
+with Git, we'd rather have you doing the former.
+
+
+### Philosophy
+
+Data Carpentry for Biologists is an open source project, and we welcome 
+contributions of all kinds: new and improved lessons, bug reports, and small 
+fixes to existing material are all useful. Course materials are managed on 
+GitHub to facilitate collaboration on developing this kind of material for 
+university courses. The central component of a [flipped computing course]({{ site.baseurl }}/documents/course-delivery) is the 
+exercises, so one of the primary form of contribution we expect will be adding 
+exercises to the existing set. Individual instructors can then select from a 
+rich pool of exercises the set that best fit the topics, languages, and 
+scientific domains they want to cover in the course.
+
+There are lots of great resources for being introduced to the individual
+concepts being taught in courses like this. Our philosophy is to use and improve
+these external resources when available instead of creating new versions of the
+same content. In particularly we actively use
+[Data Carpentry](http://datacarpentry.org/lessons) and
+[Software Carpentry](http://software-carpentry.org/lessons.html) workshop
+materials. However, in cases where the necessary material doesn't exist
+elsewhere it can certainly be added with the `materials/`.
+
+By contributing, you are agreeing that Data Carpentry may redistribute your work
+under [these licenses](LICENSE.md).
```

Comment:
"Data Carpentry may redistribute your work under [these licenses](LICENSE.md)."
->
"your work is [licensed using a combination of CC-BY and MIT licenses](LICENSE.md) and may be openly used, modified, and distributed by others."